app/components/project-geometries/draw-lots-to-union.js

```diff
@@ -80,6 +88,9 @@ export default class DrawLotsToUnion extends Component {
   @argument
   mode;
 
+  @argument
+  editableLabel = false;
```

Comment:
I wonder if we could _compose_ this type of thing by splitting it out into its own component and injecting it where it's needed (zoning-related geometries)

Reply:
Isn't that more complicated that just setting the boolean for the 3 geom types that need it?